test/e2e/tests/account/import-flow.spec.ts

```diff
@@ -9,11 +9,48 @@ import HomePage from '../../page-objects/pages/home/homepage';
 import { loginWithoutBalanceValidation } from '../../page-objects/flows/login.flow';
 import { completeImportSRPOnboardingFlow } from '../../page-objects/flows/onboarding.flow';
 
+
+async function mockAccountsApi(mockServer) {
+  return [
+    await mockServer
+      .forGet('https://accounts.api.cx.metamask.io/v2/accounts/0x0961ca10d49b9b8e371aa0bcf77fe5730b18f2e4/balances')
+      .thenCallback(() => {
+        return {
+          statusCode: 200,
+          json: {
+            count: 0,
+            balances: [],
+            unprocessedNetworks: [],
+          },
+        };
+      }),
+    await mockServer
+      .forGet('https://accounts.api.cx.metamask.io/v1/accounts/0x0961ca10d49b9b8e371aa0bcf77fe5730b18f2e4/transactions')
+      .thenCallback(() => {
+        return {
+          statusCode: 200,
+          json: {
+            data: [],
+            unprocessedNetworks: [],
+            pageInfo: {
+              hasNextPage: false,
+              cursor: null,
+              count: 0,
+            }
+          },
+        };
+      }),
+  ]
```

Comment:
these mocks correspond to the imported account, so they are added as specific mock